plugins/package-managers/gradle-inspector/README.md

```diff
@@ -31,11 +31,13 @@ Execution of the Gradle plugin will be blocked until the remote debugger is atta
 
 ## Limitations
 
-The retrieval of the checksum values for remote artifacts is currently done via plain OkHttp calls, which means it will not work out of the box for private repositories.
-To work around this, credentials need to be configured in `.netrc` additionally to in Gradle.
-This is similar to how the "legacy" [Gradle] analyzer required to additionally configure credentials in Maven.
+Currently, the [GradleInspector] has the following known limitations:
 
-Also, the `isModified` check which compares with artifacts of the same name in Maven Central is not implemented yet.
+* The retrieval of the checksum values for remote artifacts is currently done via plain OkHttp calls, which means it will not work out of the box for private repositories.
+  To work around this, credentials need to be configured in `.netrc` additionally to in Gradle.
+  This is similar to how the "legacy" [Gradle] analyzer required to additionally configure credentials in Maven.
+* The `isModified` check which compares with artifacts of the same name in Maven Central is not implemented yet.
```

Comment:
Is there something missing in "which compares with artifacts" ?

Reply:
I've clarified it.